pallets/kensetsu/src/tests.rs

```diff
@@ -95,6 +96,24 @@ fn test_create_cdp_overflow_error() {
     });
 }
 
+#[test]
+fn test_create_cdp_collateral_below_minimal() {
+    new_test_ext().execute_with(|| {
+        let minimal_balance = balance!(100);
+        set_xor_as_collateral_type(
+            Balance::MAX,
+            Perbill::from_percent(50),
+            FixedU128::from_float(0.0),
+            minimal_balance,
+        );
+
+        assert_noop!(
+            KensetsuPallet::create_cdp(alice(), XOR, balance!(0), balance!(0)),
```

Comment:
Check also that `balance!(100)` (equal with minimal balance) is ok

Reply:
Tested in `test_create_cdp_sunny_day`. I changed `minimal_collateral_deposit` in the test.